spec/amqp.spec.js

```diff
@@ -497,6 +497,50 @@ describe('AMQP', () => {
         expect(publishParameters[3]).toEqual(props);
     });
 
+    it('Should send objectId instead of message when process error', () => {
```

Comment:
Please, add integration test too in `run.spec.js`. It will help you to identify that your solution is not working.